go.mod

```diff
@@ -11,6 +11,7 @@ replace (
 	github.com/knative/pkg => github.com/rancher/pkg v0.0.0-20190514055449-b30ab9de040e
 	github.com/matryer/moq => github.com/rancher/moq v0.0.0-20200712062324-13d1f37d2d77
 	github.com/opencontainers/runc => github.com/opencontainers/runc v1.0.0-rc10
+	github.com/rancher/aks-operator => github.com/cmurphy/aks-operator v0.0.0-20210525201153-687a9fd5a1fb // FIXME
```

Comment:
This can be tagged now right ?

Reply:
It was pushed `v1.0.1-rc7`

Reply:
I pushed also change to system-charts https://github.com/rancher/system-charts/pull/479